apps/svg-preview/svg-preview.html

```diff
@@ -0,0 +1,31 @@
+<html>
+<head>
+  <title>Bliss SVG Previewer</title>
+  <style>
+    svg {
+      border: 2px solid gray;
+      background: GhostWhite;
+    }
+  </style>
+</head>
+<body>
+  <script type="module">
+    import "./svg-preview.ts";
+  </script>
+  <h1>Bliss SVG Previewer</h1>
+  <p>
+    The encoding string entered below is a space separated sequence of BCI-AV
+    identfiers and separators such as "/" and ";".  The sequence can be as
```

Comment:
Would be helpful to explain what "/" and ";" mean. Something like:

* In the format of [12335, "/", 8499], the Bliss character 12335 and the Bliss character of 8499 are displayed side by side
* In the format of [12335, ";", 8499], the Bliss character 12335 and the indicator 8499 are displayed in the way that the indicator 8499 is on top of the the charactor 12355

Reply:
Good point.  However, in exploring the Bliss SVG builder for more information, I discovered that this string will work as well:  "HC8N:0,8;S4:2,10".  It is used by the BMW palette for the first part of "Abs. Time", the "Abs." part.

That means that the legitimate encodings can be complicated and use letters, colons, and commas as well as slash and semi-colon.  I have adjusted the code to take that all into account.

However, I can't find any documentation for the structure of these encodings.  Do you know if there is any?

Reply:
I don't know about the documentation. Hannes will know better.